dbms/src/Debug/dbgFuncMisc.h

```diff
@@ -26,4 +26,9 @@ class Context;
 //   ./storage-client.sh "DBGInvoke search_log_for_key(key)"
 void dbgFuncSearchLogForKey(Context & context, const ASTs & args, DBGInvoker::Printer output);
 
+// Trigger the gc process of global storage pool. Used to remove obsolete entry left by previous dropped table
```

Comment:
```suggestion
// Trigger the gc process of global storage pool. Used to remove obsolete entries left by the previous dropped table
// Note that it is ok for now since we don't store external/ref on GlobalStoragePool.meta, or it may run into same
// problem as https://github.com/pingcap/tiflash/pull/4850
```